cfe_internal/update/update_processes.cf

```diff
@@ -154,12 +162,17 @@ bundle agent maintain_cfe_hub_process
       handle => "cfe_internal_maintain_cfe_hub_process_processes_postgres",
       ifvarclass => "nova|enterprise";
 
-    am_policy_hub.files_ok.!windows::
+    am_policy_hub.!enable_cfengine_enterprise_hub_ha.files_ok.!windows|ha_run_hub_process::
```

Comment:
We could probably use an improved comment here. Under what conditions is it important to make sure the cf-hub process is running and why?